database/gormdb_course_test.go

```diff
@@ -357,3 +357,51 @@ func TestGormDBCourseUniqueContraint(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetCourseTeachers(t *testing.T) {
+	tests := map[string]struct {
+		wantTeachers, students []*pb.User
+	}{
+		"Basic": {
+			wantTeachers: []*pb.User{{Login: "teacher1"}, {Login: "teacher2"}},
+			students:     []*pb.User{{Login: "student1"}},
+		},
+		"No teachers": {
```

Comment:
I think this should expect an error and not an empty `User` slice because the `CreateCourse` method requires a user id / course creator and said user is enrolled as teacher. Hence `wantTeachers` should be `nil`. There should be an additional field here to check for the expected error.